src/SettingsRepositories/DatabaseSettingsRepository.php

```diff
@@ -71,6 +73,25 @@ public function updatePropertyPayload(string $group, string $name, $value): void
             ]);
     }
 
+    public function updatePropertiesPayload(string $group, Collection $properties): void
```

Comment:
Since this will be a major release, could we drop `updatePropertyPayload` altogether to make a unified update path in the code?

Reply:
Done